src/aws_credentials_file.erl

```diff
@@ -67,5 +67,6 @@ parse_file(Path, Options) ->
         Profile ->
             {ok, aws_credentials:make_map(?MODULE,
                                           maps:get(<<"aws_access_key_id">>, Profile),
-                                          maps:get(<<"aws_secret_access_key">>, Profile)), infinity}
+                                          maps:get(<<"aws_secret_access_key">>, Profile),
+                                          maps:get(<<"aws_session_token">>, Profile, "")), infinity}
```

Comment:
Would prefer something like
```erlang
case maps:is_key(<<"aws_session_token">>, Profile) of
   true -> 
    {ok, aws_credentials:make_map(?MODULE,
                                          maps:get(<<"aws_access_key_id">>, Profile),
                                          maps:get(<<"aws_secret_access_key">>, Profile),
                                          maps:get(<<"aws_session_token">>, Profile), infinity};
   false ->
      {ok, aws_credentials:make_map(?MODULE,
                                          maps:get(<<"aws_access_key_id">>, Profile),
                                          maps:get(<<"aws_secret_access_key">>, Profile), infinity)
end
```
I'm sure there's prettier code but... that's the idea, check if it's there, include if so, omit if not

Reply:
Updated - thanks!